### PR TITLE
[backend/worker] Logs improvement for python execution and worker

### LINKED ISSUE #5413: Logs improvement for python execution and worker
## Use case

Logs improvement for python execution and worker

### PR DESCRIPTION
**Pull request by richard-julien**
[[backend/worker] Logs improvement for python execution and worker](https://github.com/OpenCTI-Platform/opencti/commit/a359862c864050fd0e8c1e8d67c325f79a099b27)

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/python/pythonBridge.js [hunk 1]
```diff
@@ -111,29 +111,25 @@ const checkChildPythonAvailability = async (context, user) => {
...
 const execNativePython = async (context, user, script, ...args) => {
   const execNativePythonFn = async () => {
-    try {
-      const result = py.callSync(script.py, script.fn, ...args);
-      if (result.status === 'success') {
-        return result.data;
-      }
-      throw UnknownError('[BRIDGE] execNativePython error', { detail: result.data });
-    } catch (err) {
-      throw UnknownError(`[BRIDGE] execNativePython error > ${err.message}`, { detail: err.message });
```
Comment (helene-nguyen) [inline opencti-platform/opencti-graphql/src/python/pythonBridge.js:121]: Why remove try catch ?

File: opencti-platform/opencti-graphql/src/python/pythonBridge.js [hunk 2]
```diff
@@ -21,6 +22,9 @@ const CREATE_PATTERN_SCRIPT = { fn: 'stix2_create_pattern', py: pyCreatePattern
 // region child
 export const execChildPython = async (context, user, scriptPath, scriptName, args, stopCondition) => {
   const execPythonTestingProcessFn = async () => {
+    if (isEmptyField(scriptPath) || isEmptyField(scriptName)) {
+      throw UnsupportedError('Cant exec python with empty script path or name');
```
Comment (helene-nguyen) [inline opencti-platform/opencti-graphql/src/python/pythonBridge.js]:
```suggestion
      throw UnsupportedError('Cannot execute Python with empty script path or name');
```

File: opencti-platform/opencti-graphql/tests/01-unit/python/pythonBridge-test.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ it('Check if python is well configured', async () => {
   expect(check).not.toBeNull();
   expect(check).toEqual("[text:value = 'test']");
   // noinspection ES6MissingAwait
-  expect(execChildPython(testContext, ADMIN_USER, '/missing')).rejects.toThrow('[BRIDGE] execPythonTesting error');
+  expect(execChildPython(testContext, ADMIN_USER, '/missing')).rejects.toThrow('Cant exec python with empty script path or name');
```
Comment (helene-nguyen) [inline opencti-platform/opencti-graphql/tests/01-unit/python/pythonBridge-test.js]:
```suggestion
  expect(execChildPython(testContext, ADMIN_USER, '/missing')).rejects.toThrow('Cannot execute Python with empty script path or name');
```

File: opencti-platform/opencti-graphql/src/python/pythonBridge.js (hunk 1 above)
Comment (richard-julien) [reply opencti-platform/opencti-graphql/src/python/pythonBridge.js:121]: That was unneeded and prevent to have the  correct amount of information

**Pull request merged**